sdk-tests/src/test/java/io/dapr/it/DaprRun.java

```diff
@@ -170,16 +171,17 @@ public String getAppName() {
   }
 
   private static String buildDaprCommand(
-      String appName, Class serviceClass, DaprPorts ports, DaprApiProtocol protocol) {
-    StringBuilder stringBuilder = new StringBuilder(String.format(DAPR_RUN, appName))
-        .append(ports.getAppPort() != null ? " --app-port " + ports.getAppPort() : "")
-        .append(ports.getHttpPort() != null ? " --dapr-http-port " + ports.getHttpPort() : "")
-        .append(ports.getGrpcPort() != null ? " --dapr-grpc-port " + ports.getGrpcPort() : "")
-        .append(serviceClass == null ? "" :
-            String.format(DAPR_COMMAND, serviceClass.getCanonicalName(),
-                ports.getAppPort() != null ? ports.getAppPort().toString() : "",
-                Properties.API_PROTOCOL.getName(), protocol,
-                Properties.API_METHOD_INVOCATION_PROTOCOL.getName(), protocol));
+      String appName, Class serviceClass, DaprPorts ports, DaprApiProtocol protocol, DaprApiProtocol appProtocol) {
+    StringBuilder stringBuilder =
+        new StringBuilder(String.format(DAPR_RUN, appName, appProtocol.toString().toLowerCase()))
+            .append(ports.getAppPort() != null ? " --app-port " + ports.getAppPort() : "")
+            .append(ports.getHttpPort() != null ? " --dapr-http-port " + ports.getHttpPort() : "")
+            .append(ports.getGrpcPort() != null ? " --dapr-grpc-port " + ports.getGrpcPort() : "")
+            .append(serviceClass == null ? "" :
+                String.format(DAPR_COMMAND, serviceClass.getCanonicalName(),
+                    ports.getAppPort() != null ? ports.getAppPort().toString() : "",
+                    Properties.API_PROTOCOL.getName(), protocol,
+                    Properties.API_METHOD_INVOCATION_PROTOCOL.getName(), protocol));
```

Comment:
should this be appProtocol?

Reply:
No, this is for the method API protocol, which is which protocol we use when invoking a service.